boards/renesas/rzg3s_smarc/rzg3s_smarc_r9a08g045s33gbg_cm33_defconfig

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2024 EPAM Systems
+# Copyright (c) 2024 Renesas Electronics Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SYS_CLOCK_EXISTS=y
+
+CONFIG_XIP=n
+
+CONFIG_MAIN_STACK_SIZE=4096
+
+# Enable UART driver
+CONFIG_SERIAL=y
+
+# Enable console
+CONFIG_CONSOLE=y
+CONFIG_UART_CONSOLE=y
+
+# This is required to support debug with xSPI boot when Zephyr already booted by TF-A
+CONFIG_INIT_ARCH_HW_AT_BOOT=y
+
+CONFIG_BUILD_OUTPUT_S19=y
```

Comment:
should be set at soc level

Reply:
Thank you.
I moved them to `soc\renesas\rz\rzg3s\Kconfig.defconfig`